src/main/resources/config_file_documentation.txt

```diff
@@ -149,7 +149,7 @@ Config file must be valid JSON format. Examples can be found under the test fold
         default: false
 
       "overvoteLabel" optional
-        label used in the CVR to denote an overvote; if this parameter is present overvoteRule must be either "alwaysSkipToNextRank" or "exhaustImmediately" (because the other options rely on knowing which specific candidates were involved in each overvote)
+        label used in the CVR to denote an overvote; if this parameter is present overvoteRule must be either "alwaysSkipToNextRank" or "exhaustImmediately" (because the other option, exhaustIfMultipleContinuing, relies on knowing which specific candidates were involved in each overvote)
```

Comment:
Also need to fix "overvoteRule" section in doc above to eliminate the other options.